src/app/ui/components/privacy/private-text.layout.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Maybe we can introduce a `container` component here - `PrivateText`, so that it can fetch the setting from the state and avoid drilling it down?

So it would be something like `private-text.tsx` 
`
import { PrivateTextLayout } from '@app/ui/components/privacy/private-text.layout'; 

export function PrivateText() {

const isPrivate = useIsPrivateMode();

return (
<PrivateTextLayout
                isPrivate={isBalancePrivate}
                onShowValue={onShowBalance}
                display="inline-block"
                overflow="hidden"
              >
                {balance}
              </PrivateTextLayout>
)
}
```

That could help avoid having to drill down `isPrivate={isPrivate}` so often and we could isolate it into it's own feature: `features/privacy/`. We could also add `onShowValue={onShowBalance}` in there

Reply:
OK great, thanks for the explanation 👍